File: src/models/sound.ts
export class Sound {
    public id: number;
    public title: string;
    public author: string;
    public liked: number;
    public soundUrl: string;

    /**
     *
     */
    constructor(id: number, title: string, author: string, liked: number, soundUrl: string) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.liked = liked;
        this.soundUrl = soundUrl;
    }

}

File: server/DataService/Models/Entities/Sound.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataService.Models.Entities
{
    public partial class Sound
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public int Liked { get; set; }
        public string SoundUrl { get; set; }
    }
}
